docs/pages/Architecture/Advertisement Management/broadcaster.md

```diff
@@ -28,12 +28,12 @@ After having created the Advertisement on the remote cluster, the broadcaster st
    the new amount of resources and immediately pushes it on the foreign cluster, without waiting for the periodic creation.
 
 ### Limitations
-* More complex policies (e.g. differentiate Advertisement on the base of the foreign cluster).
-* MetricsAPI to have more precise values of available resources.
+* Implement more complex policies (e.g. differentiate Advertisement on the base of the foreign cluster).
+* Read data from [MetricsAPI](https://kubernetes.io/docs/tasks/debug-application-cluster/resource-metrics-pipeline/) to have more precise values of resources currently used by pods and a better computation of available resources.
 
 ## Architecture and workflow
 
-![](/images/advertisement-protocol/broadcaster-workflow.png)
+![workflow](/images/advertisement-protocol/broadcaster-workflow.png) 
```

Comment:
I don't know why, all the images in the `Architecture/Advertisement Management` chapter are not displayed. I tried to add/remove the description but nothing changes, searched this issue but couldn't find a solution. I don't know if it could be a problem related to the image dimensions

Reply:
I think it should be related to the space in the name.

Reply:
which space? the one in the name of the folder? also `Discovery and Peering` folder has spaces but the images are correctly displayed

Reply:
uhm you are right, I check this.

Reply:
It seems to be somewhat related to ublock/ublocker. The images with relative paths (e.g. the one of the dashboard) work correcly